packages/devtools_app/lib/src/config_specific/import_export/import_export.dart

```diff
@@ -51,9 +51,9 @@ class ImportController {
 
   final void Function(String screenId) _pushSnapshotScreenForImport;
 
-  final NotificationService _notifications;
+  final NotificationService? _notifications;
```

Comment:
make this non-nullable since we just unwrap it everywhere it is used here

Reply:
done

Reply:
I did this and scaffold_test.dart started failing because the constructor Notifications.of(context) returns null. 
Is there a way to update the test so that `context.dependOnInheritedWidgetOfExactType<_InheritedNotifications>();` returns non null?

Reply:
Yes, try using the wrapWithNotifications helper. It is used in a few other tests so there are some examples.